src/dom/node-manager/default-input-node.js

```diff
@@ -22,6 +22,9 @@ export default BaseView.extend({
             const changeHandler = function () {
                 const val = this.getUIValue();
                 const payload = [{ name: propName, value: val }];
+                if (val.length === 0) {
+                    return this.$el.trigger(events.trigger, { data: payload, source: 'bind', restoreOriginal: true });
```

Comment:
Any reason we're not just adding `restoreOrigina: val.length === 0` to the original call rather than adding an if block?  Not a big deal either way, just curious.

Reply:
Changed